charts/bulk-scan-orchestrator/values.template.yaml

```diff
@@ -0,0 +1,32 @@
+java:
+  applicationPort: 8582
+  readinessPath: '/health'
+  livenessPath: '/health'
+  secrets:
+    SB_CONN_STRING:
+      secretRef: servicebus-secret-namespace-${SERVICE_NAME}
+      key: connectionString
+  environment:
+    S2S_SECRET: "${S2S_SECRET}"
+    IDAM_CLIENT_SECRET: "${IDAM_CLIENT_SECRET}"
+    IDAM_USER_NAME: "${IDAM_USER_NAME}"
```

Comment:
Is IDAM_USER_NAME and IDAM_USERS_BULKSCAN_USERNAME needed?

Reply:
It was when I wrote the k8s template.

Reply:
Its used in the functional tests not in the application, from a quick grep i just did

Reply:
i.e. IDAM_USER_NAME is tests only the bulkscan one is required